src/test/java/com/google/jenkins/plugins/computeengine/integration/ITUtil.java

```diff
@@ -76,24 +77,18 @@
 import java.util.Map;
 import java.util.logging.Logger;
 import jenkins.util.SystemProperties;
+import lombok.extern.java.Log;
 import org.apache.commons.lang.SystemUtils;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
 import org.jvnet.hudson.test.JenkinsRule;
 
 /** Common logic and constants used throughout the integration tests. */
+@Log
 class ITUtil {
     static final boolean windows =
             Boolean.parseBoolean(SystemProperties.getString(ITUtil.class.getName() + ".windows", "false"));
     static final boolean customssh =
             Boolean.parseBoolean(SystemProperties.getString(ITUtil.class.getName() + ".customssh", "false"));
-    private static final String DEB_JAVA_STARTUP_SCRIPT = "#!/bin/bash\n"
-            + "/etc/init.d/ssh stop\n"
-            + "echo \"deb http://http.debian.net/debian stretch-backports main\" | \\\n"
-            + "      sudo tee --append /etc/apt/sources.list > /dev/null\n"
-            + "apt-get -y update\n"
-            + "apt-get -y install -t stretch-backports openjdk-8-jdk\n"
-            + "update-java-alternatives -s java-1.8.0-openjdk-amd64\n"
-            + "/etc/init.d/ssh start";
-
     static final String PROJECT_ID = System.getenv("GOOGLE_PROJECT_ID");
```

Comment:
need to do further work to automatically detect the default project if the environment variable is not given.

It is still good to have environment variable this though, so people working on multiple projects can switch easily in test.

Reply:
> It is still good to have environment variable this though, so people working on multiple projects can switch easily in test.

Why not just `gcloud config configurations activate`?